app/components/Review/fields/ArrayBooleanField.tsx

```diff
@@ -0,0 +1,38 @@
+import { FieldProps } from '@rjsf/core';
+import React from 'react';
+import { StyledColLeft, StyledColRight, StyledColError } from '../index';
+
+// Field to display a list of yes/no values instead of the enum for array type fields
```

Comment:
```suggestion
/**
Field to display a list of yes/no values instead of the enum for array type fields
*/
```
consider using jsdoc so that VSCode displays the comment when hovering on the name